README.md

```diff
@@ -2432,6 +2432,37 @@ config.properties['hazelcast.client.statistics.period.seconds'] = 4;
 
 After enabling the client statistics, you can monitor your clients using Hazelcast Management Center. Please refer to the [Monitoring Clients section](https://docs.hazelcast.org/docs/management-center/latest/manual/html/index.html#monitoring-clients) in the Hazelcast Management Center Reference Manual for more information on the client statistics.
 
+### 7.9.2. Logging Configuration
+
+Hazelcast Node.js client has a flexible logging configuration. You can add your custom loggers by providing adapters.
+
+To configure the logger, you need to implement `ILogger` interface in your adapter and add it to `ClientConfig.properties['hazelcast.logging']`. If you set this property to `'off'`, it does not log anything.
```

Comment:
To configure a logger, you need to implement the `ILogger` interface ...

NOTE: I am not sure if my question is reasonable but: Don't we need to show a brief code example on how to implement the ILogger interface and pass it to the properties?

Reply:
I think the below code snippet shows it.